src/platform/windows/misc.cpp

```diff
@@ -464,6 +466,37 @@ bp::child run_unprivileged(const std::string &cmd, boost::filesystem::path &work
       (LPSTARTUPINFOW)&startup_info,
       &process_info);
 
+    if(!ret) {
+      auto error = GetLastError();
+
+
+      if(error == 740) {
+        BOOST_LOG(info) << "Could not execute previous command because it required elevation. Running the command again with elevation, for security reasons this will prompt user interaction."sv;
+        startup_info.StartupInfo.wShowWindow = SW_HIDE;
+        startup_info.StartupInfo.dwFlags     = startup_info.StartupInfo.dwFlags | STARTF_USESHOWWINDOW;
+        std::wstring elevated_command        = L"tools\\elevator.exe ";
+        elevated_command += wcmd;
+
+        // For security reasons, Windows enforces that an application can only have one "interactive thread," responsible for processing input from the user and managing the user interface (UI).
+        // Since UAC prompts are interactive, we can't have a UAC prompt while Sunshine is already running because it would block the thread.
+        // To workaround this, we launch a separate process that will prompt the user for elevation and then launch the actual command.
```

Comment:
`work around`

Reply:
Huh? Are you asking me to break the word "workaround"?

Workaround is a valid word in the dictionary in this use case.

Reply:
It's not a big deal, but "workaround" is a noun, "work around" is a verb.

Reply:
I wouldn't have even pick up on this if it weren't for my IDE.